core/domain/user_domain.py

```diff
@@ -282,6 +282,67 @@ def __init__(self, user_id, score_category, score, has_email_been_sent):
         self.score = score
         self.has_email_been_sent = has_email_been_sent
```

Comment:
Could you please change this, in both the domain and storage models, to proficiency_email_sent? We try not to start variable/attr names with a verb, and there needs to be a bit more explanation of what email this refers to.

Normally, we cannot do this, but fortunately no storage models exist yet in prod, so it's safe.

Reply:
Also -- what do you think about changing the name of this class to UserContributionProficiency (and similarly, UserContributionProficiencyModel for the storage model)? Again, this is only possible since this isn't used in prod yet since the dashboard isn't launched. The current name seems a bit off since there's more to this class than "scoring".